ansible/roles/kubernetes-addons/tasks/cluster-monitoring.yml

```diff
@@ -13,6 +13,7 @@
     url=https://raw.githubusercontent.com/GoogleCloudPlatform/kubernetes/master/cluster/addons/cluster-monitoring/influxdb/{{ item }}
     dest="{{ local_temp_addon_dir }}/cluster-monitoring/{{ item }}.j2"
     force=yes
+    validate_certs=False
```

Comment:
It looks like you found a bug, where some add ons are missing this. I agree it's needed. For the purposes of keeping the PR clean, add notes to the PR description for changes above and beyond Contiv support. This helps if we need to revert, etc.. for PR's to match what changes they make.